cookbooks/masakari/attributes/default.rb

```diff
@@ -0,0 +1,2 @@
+default['masakari_ci_conf']['masakari_git'] = "https://github.com/ntt-sic/masakari.git"
```

Comment:
I'm not confident on my comment, though, does 'masakari_ci_conf' come from other recipe? why is it not 'masakari' rather than 'masakari_ci_conf'?